astropy/io/ascii/latex.py

```diff
@@ -75,19 +75,20 @@ def find_latex_line(lines, latex):
 
 
 class LatexSplitter(core.BaseSplitter):
-    '''Split LaTeX table date. Default delimiter is `&`.
+    '''Split LaTeX table data. Default delimiter is `&`.
     '''
     delimiter = '&'
 
-    def process_line(self, line):
+    def process_line(self, line, last_line=False):
         """Remove whitespace at the beginning or end of line. Also remove
         \\ at end of line"""
         line = line.split('%')[0]
         line = line.strip()
         if line[-2:] == r'\\':
             line = line.strip(r'\\')
         else:
-            raise core.InconsistentTableError(r'Lines in LaTeX table have to end with \\')
+            if not last_line:
+                raise core.InconsistentTableError(r'Lines in LaTeX table have to end with \\')
```

Comment:
This is also missing a `.` at the end of the sentence. Also this looks quite longish, maybe split it into two lines?

```
                raise core.InconsistentTableError(
                    r'Lines in LaTeX table have to end with \\.')
```